sdk/keyvault/keyvault-certificates/CHANGELOG.md

```diff
@@ -1,7 +1,13 @@
 # Release History
 
-## 4.1.0-preview.2 (Unreleased)
+## 4.1.0 (Unreleased)
 
+- Added the optional `serviceVersion` property to the `CertificateClient` optional parameters.
+  It defaults to the latest supported API version, which currently is `7.1`.
+- Added `recoverableDays` as an optional property to `KeyProperties`.
+- Fixed a bug on `importCertificate`. The bytes of a PEM formatted certificate were being encoded again on base64.
+  Now, if the user provides a policy with contentType `application/x-pem-file`, the bytes will be assumed to be formatted using ASCII,
+  and thus will be sent as is.
```

Comment:
The 3 bug fixes listed below were part of 4.0.2 release for certificates. So, they should not be appearing again for 4.1.0

Reply:
Thank you for your feedback! 🌞

I have copied to 4.1.0 only the changes that have happened after the last non-preview version. I have also restored the existing information, but I realized that I forgot to add to master the changelog entries for the last non-preview versions, so I have added those.

Feedback appreciated!